app/controllers/support_interface/uploads_controller.rb

```diff
@@ -41,6 +46,10 @@ def cancel
 
     private
 
+    def clear_failed_entries
+      FailedChildrensBarredListEntries.new.clear(@upload_file_hash)
```

Comment:
I think I've been consistently inconsistent here.
It could use the `upload_file_hash_param` method as suggested but I think the before callback needs to omit `create` as well. Will fix.